examples/update_header_keywords.py

```diff
@@ -1,20 +1,22 @@
 """
-==========================================================
-Updating pointing and observer keywords in the FITS header
-==========================================================
+=============================================
+Updating pointing keywords in the FITS header
```

Comment:
Wondering if this example is still useful now?

Reply:
The pointing update bit is still useful I think. Whether it needs its own example I'll leave up to you.